src/uct/ib/mlx5/rc/rc_mlx5_iface.c

```diff
@@ -647,6 +646,7 @@ uct_rc_mlx5_iface_is_reachable_v2(const uct_iface_h tl_iface,
     uint8_t my_type = uct_rc_mlx5_iface_get_address_type(tl_iface);
     uint8_t remote_type;
     const uct_iface_addr_t *iface_addr;
+    static const char *tm_type_to_str[] = {"basic", "tag matching"};
```

Comment:
pls move it before my_type since initialized vars should be before uninitialized ones